config/HOWTO.md

```diff
@@ -67,4 +67,28 @@ You should also double check the credentials which are used to perform the API c
 
 ## Header integration 
 
-[TODO]
+In order to add the geOcrhestra header, youy can use the [webcomponent `geor-header`](https://github.com/georchestra/header/) and add it to the `index.html` file. 
```

Comment:
In order to add the geOrchestra header to the maelstro application, one can use the `geor-header` [webcomponent](https://github.com/georchestra/header/) and add it to the `index.html` file.